services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchSyncSpec.groovy

```diff
@@ -214,6 +215,33 @@ class SwitchSyncSpec extends BaseSpecification {
         flowHelper.deleteFlow(flow.id)
     }
 
+    def "System doesn't synchronize missing default rules"() {
+        given: "An active switch with missing rules"
+        def sw = topology.activeSwitches.first()
+        def defaultRules = northbound.getSwitchRules(sw.dpId).flowEntries
+        northbound.deleteSwitchRules(sw.dpId, DeleteRulesAction.DROP_ALL)
+        def initRules = northbound.validateSwitchRules(sw.dpId)
+        def amountProperRules = initRules.properRules.size()
+        def amountMissingRules = initRules.missingRules.size()
+        assert defaultRules.size() == amountMissingRules
+        def amountExcessRules = initRules.excessRules.size()
+
+        when: "Synchronize rules on the switch"
+        northbound.synchronizeSwitchRules(sw.dpId)
+
+        then: "Missing rules are not synchronized"
+        def rulesAfterSync = northbound.validateSwitchRules(sw.dpId)
+        amountProperRules == rulesAfterSync.properRules.size()
+        amountMissingRules == rulesAfterSync.missingRules.size()
+        amountExcessRules == rulesAfterSync.excessRules.size()
+
```

Comment:
Why `defaultRules`, `amountProperRules`, `amountMissingRules`, `amountExcessRules`, `initRules` are required in this test?
just do
```
then: "Missing rules are not synchronized"
with(northbound.validateSwitchRules(sw.dpId)) {
   properRules.empty
   excessRules.empty
   missingRules.sort() == sw.defaultCookies.sort()
}
```
that's it

Reply:
ok, fixed